dbms/src/Flash/Coprocessor/DAGQueryInfo.h

```diff
@@ -39,6 +41,7 @@ struct DAGQueryInfo
         , filters(filters_)
         , ann_query_info(ann_query_info_)
         , pushed_down_filters(pushed_down_filters_)
+        , used_indexes(DM::PBToLocalIndexInfos(used_indexes_))
```

Comment:
Maybe if you simply pass around `tipb::ColumnarIndexInfo` then you don't need `DM::PBToLocalIndexInfos` at all?

Reply:
`LocalIndexInfo` is a better in-memory data struct.